oci/oci.go

```diff
@@ -486,6 +486,9 @@ func (r *Runtime) UpdateStatus(c *Container) error {
 	defer c.opLock.Unlock()
 	out, err := exec.Command(r.Path(c), "state", c.name).CombinedOutput()
 	if err != nil {
+		if strings.Contains(string(out), "does not exist") {
```

Comment:
Instead of pinging the process (we may have not a PID at all at this point, after a reboot for instance), can't we just say:
"if `runc state` fails for whatever reason, then set the container as stopped as below and return `nil`". @sameo how does that sound?